editor/src/components/canvas/controls/select-mode/cursor-overlay.tsx

```diff
@@ -18,56 +18,30 @@ export function getCursorForOverlay(editorState: EditorState): CSSCursor | null
 
 export const CursorOverlay = React.memo(() => {
   const cursor = useDelayedEditorState((store) => {
-    return getCursorForOverlay(store.editor)
+    return getCursorFromEditor(store.editor)
   })
 
-  const imageDragSessionInProgress =
-    useDelayedEditorState((store) => store.editor.imageDragSessionState.type !== 'NOT_DRAGGING') ??
-    false
-
-  const styleProps = React.useMemo(() => {
-    let workingStyleProps: React.CSSProperties = {
-      position: 'fixed',
-      left: 0,
-      top: 0,
-      width: '100vw',
-      height: '100vh',
-      pointerEvents: pointerEventsFromWorldState({
-        isCursorSet: cursor != null,
-        imageDragInProgress: imageDragSessionInProgress,
-      }),
-      zIndex: 9999999,
-    }
+  React.useEffect(() => {
+    const editor = document.getElementById(EditorID)
     if (cursor != null) {
-      workingStyleProps.cursor = cursor
+      document.body.classList.add('customCursor')
+      document.body.style.cursor = cursor
+    } else {
+      document.body.classList.remove('customCursor')
+      document.body.style.cursor = 'unset'
+    }
+
+    if (editor != null) {
+      editor.style.pointerEvents = 'initial'
```

Comment:
This could just go in the `index.html` unless there's a reason to set it via the `useEffect`

Reply:
I want to avoid deleting it accidentally

Reply:
I don't understand?

Reply:
well, I was thinking that we frequently change inline styles and it's easy to break this, but I realized that we rarely touch the html, so I moved it there now